docs/development/build.md

```diff
@@ -44,10 +44,36 @@ To enable `libimpctl` support `GO_FLAGS` variable must be set:
 $GOPATH/src/github.com/google/cadvisor $ GO_FLAGS="-tags=libipmctl,netgo" make build
 ```
 
+### Perf Support
+
+cAdvisor can be linked against [libpfm4](http://perfmon2.sourceforge.net/) library that allows to gather information about performance monitoring events.
+If you want to build cAdvisor with libpfm4 support you must meet following requirements:
+* `libpfm4-dev` must be installed on build system.
+* `libpfm4` must be installed on all systems where cAdvisor is running.
+
+libpfm4 can be installed using apt package manager:
+```
+apt-get install libpfm4 libpfm4-dev
```

Comment:
I think that it would be worth mentioning the libpfm4 packages are available in both: Debian- and RHEL-derivatives.